pkg/kopia/snapshot/snapshot.go

```diff
@@ -67,10 +67,11 @@ func SnapshotSource(
 		return "", 0, errors.Wrap(err, "Failed to save kopia manifest")
 	}
 
-	_, err = policy.ApplyRetentionPolicy(ctx, rep, sourceInfo, true)
-	if err != nil {
-		return "", 0, errors.Wrap(err, "Failed to apply kopia retention policy")
-	}
+	// TODO: https://github.com/kanisterio/kanister/issues/2441
+	// _, err = policy.ApplyRetentionPolicy(ctx, rep, sourceInfo, true)
+	// if err != nil {
+	// 	return "", 0, errors.Wrap(err, "Failed to apply kopia retention policy")
+	// }
```

Comment:
Should commented code be removed, and instead be replaced with a description?

Not sure whether issue #2441 already has a link to this PR / commit. If not, this comment should create the link anyway.